spec/models/order_address_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+RSpec.describe OrderAddress, type: :model do
+    before do
+      @user = FactoryBot.create(:user)
+      @item = FactoryBot.create(:item)
+      @order_address = FactoryBot.build(:order_address, item_id: @item.id, user_id: @user.id)
+      sleep (0.1)
+    end
+
+    context '商品の購入ができる時' do
+      it '全ての項目が正しく入力されていれば購入できる' do
+        expect(@order_address).to be_valid
+      end
+      it '建物名がなくても購入できる' do
+        @order_address.building_name = ''
+        expect(@order_address).to be_valid
+      end
+    end
+
+    context '商品の購入ができない場合' do
+      it 'postal_codeが空だと保存できないこと' do
+        @order_address.postal_code = ''
+        @order_address.valid?
+        expect(@order_address.errors.full_messages).to include("Postal code can't be blank")
+      end
+      it 'postal_codeが半角のハイフンを含んだ正しい形式でないと保存できないこと' do
+        @order_address.postal_code = '1234567'
+        @order_address.valid?
+        expect(@order_address.errors.full_messages).to include('Postal code is invalid. Include hyphen(-)')
+      end
+      it 'prefecture_idを選択していないと購入できないこと' do
+        @order_address.prefecture_id = 0
```

Comment:
```suggestion
        @order_address.prefecture_id = 1
```
バリデーションと同様、0を1へ変更し、テストが正常に実行されるか確認しましょう。